src/main/scala/chisel3/LTL.scala

```diff
@@ -0,0 +1,391 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.ltl
```

Comment:
The file path should reflect the package, please put this in a new directory called `ltl`